.gitignore

```diff
@@ -9,3 +9,5 @@ perf.*
 dist/
 result
 db/
+
+/test artifacts
```

Comment:
I’d actually rather use `tempfile` for the unit tests instead of adding the `/test` artifacts to .gitignore. It keeps the repo a bit cleaner. mind adding this to the `dev-dependencies`

```
[dev-dependencies]
tempfile = "0.3"
```